lib/perimeterx.js

```diff
@@ -47,32 +47,49 @@ function pxMiddleware(req, res, next) {
             res.clearCookie('_pxCaptcha', {});
             pxCaptcha.verifyCaptcha(pxCtx, pxcptch, (err, result) => {
                 if (err) {
-                    pxLogger.error('error while evaluation perimeterx captcha');
-                    return pxBlock(pxCtx, pxconfig.SCORE_EVALUATE_ACTION.CAPTCHA_BLOCK_TRAFFIC, pxconfig.BLOCK_HANDLER, req, res, next);
+                    if (err == "timeout") {
+                        pxLogger.debug('sending page requested activity for captcha timeout');
+                        pxCtx.pass_reason = "captcha_timeout";
```

Comment:
on my opinion it will look better to put these strings into a variable
look on the example of `pxconfig.SCORE_EVALUATE_ACTION.CAPTCHA_BLOCK_TRAFFIC`
I think it will look better to do something like pxconfig.PASS_REASON.REASON